linux-anvil-aarch64/Dockerfile

```diff
@@ -53,9 +54,8 @@ RUN source /opt/conda/etc/profile.d/conda.sh && \
         $_channel::libgfortran-ng \
         $_channel::libstdcxx-ng && \
     conda remove --yes --quiet -n test --all && \
-    conda clean -tsy && \
-    chgrp -R lucky /opt/conda && \
-    chmod -R g=u /opt/conda
+    conda clean -tiy && \
+    find /opt/conda ! -group lucky -exec chgrp -h lucky {} + -exec chmod u=g {} +
```

Comment:
This runs `chgrp`/`chmod` only for those files that haven't already been `chgrp`'d for the `lucky` group => we avoid touching those files created by `scripts/run_commands` in the lower layer.